src/main/java/cn/nukkit/Server.java

```diff
@@ -1572,7 +1572,6 @@ public IPlayer getOfflinePlayer(final String name) {
     public IPlayer getOfflinePlayer(UUID uuid) {
         Preconditions.checkNotNull(uuid, "uuid");
         Optional<Player> onlinePlayer = getPlayer(uuid);
-        //noinspection OptionalIsPresent
```

Comment:
Why was this removed?

Reply:
Because IDEA said it's unnecessary, but I haven't checked deeply

Reply:
okay, I got it.
```java
Optional<Player> onlinePlayer = getPlayer(uuid);
// checks if there is value
if (onlinePlayer.isPresent()) {
  // so, the `get()` is totally safe
  return onlinePlayer.get();
}
```